packages/zosfiles/src/api/methods/copy/doc/ICopyDatasetOptions.ts

```diff
@@ -0,0 +1,21 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+/**
+ * This interface defines the options that can be sent into the copy data set function.
+ */
+export interface ICopyDatasetOptions {
```

Comment:
I believe this Interface could be extended to support all options on the payload.
However there are more changes to the payload on #599.
I'd suggest merging this in, and making all necessary updates on the other PR :)